homeassistant/components/light/fibaro.py

```diff
@@ -61,15 +64,27 @@ def __init__(self, fibaro_device, controller):
         self._color = (0, 0)
         self._brightness = None
         self._white = 0
+        devconf = fibaro_device.device_config
+        force_color = devconf.get(CONF_COLOR, None)
+        force_dim = devconf.get(CONF_DIMMING, None)
+        force_white_value = devconf.get(CONF_WHITE_VALUE, None)
+        self._reset_color = devconf.get(CONF_RESET_COLOR, False)
 
         self._update_lock = asyncio.Lock()
-        if 'levelChange' in fibaro_device.interfaces:
+        if ('levelChange' in fibaro_device.interfaces or
+                force_dim is True) and \
+                force_dim is not False:
```

Comment:
Could probably simplify this logic.
```python
if ('levelChange' in fibaro_device.interfaces or force_dim):
```
As it is right now, level_change in interfaces (True), force_dim: False would result in
`(True or False) and False` -> `True and False` -> `False`